custom_components/easee/services.py

```diff
@@ -261,32 +265,40 @@ async def charger_execute_set_circuit_current(call):
         currentP2 = call.data.get(ATTR_SET_CURRENTP2)
         currentP3 = call.data.get(ATTR_SET_CURRENTP3)
 
-        _LOGGER.debug("execute_service:" + str(call.data))
+        _LOGGER.debug("execute_service: " + str(call.service) + " " + str(call.data))
```

Comment:
Follow HA code guidelines
```suggestion
        _LOGGER.debug("execute_service: %s %s", str(call.service), str(call.data))
```

Reply:
I will probably remove the debug printout entirely...